src/commands/verify.ts

```diff
@@ -1,6 +1,6 @@
 import { Argv } from "yargs";
 import signale from "signale";
-import { VerifyCommand } from "./command-types";
+import { VerifyCommand } from "./command.types";
```

Comment:
why ?

Reply:
ok will revert

Reply:
The types files are conventionally named as foo.type.ts, but this file is named command-types.ts